binderhub/builder.py

```diff
@@ -424,9 +424,14 @@ def escape(s):
 
     async def launch(self, kube, provider):
         """Ask JupyterHub to launch the image."""
+        # Load the spec-specific configuration if it has been overridden
+        spec_config = provider.spec_configuration_override
+
         # check quota first
         if provider.has_higher_quota():
             quota = self.settings.get('per_repo_quota_higher')
+        elif "per_repo_quota" in spec_config:
+            quota = spec_config.get('per_repo_quota')
```

Comment:
```suggestion
            quota = spec_config.get('quota')
```
(also on the line above)

Though you could write this as:

```python
quota = spec_config.get('quota', self.settings.get('per_repo_quota'))
```
which is a well used pattern in Python for saying "get this from the dictionary and if it doesn't exist use this thing as default"